# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qt/X11R7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We no longer have X11R6 ... ,-)

  - Rene Rebe <rene@exactcode.de>

--- ./mkspecs/linux-g++-64/qmake.conf.vanilla	2005-10-08 09:38:19.000000000 +0200
+++ ./mkspecs/linux-g++-64/qmake.conf	2005-10-08 09:38:27.000000000 +0200
@@ -43,12 +43,12 @@
 
 QMAKE_INCDIR		=
 QMAKE_LIBDIR		=
-QMAKE_INCDIR_X11	= /usr/X11R6/include
-QMAKE_LIBDIR_X11	= /usr/X11R6/lib64
+QMAKE_INCDIR_X11	= /usr/X11/include
+QMAKE_LIBDIR_X11	= /usr/X11/lib64
 QMAKE_INCDIR_QT		= $(QTDIR)/include
 QMAKE_LIBDIR_QT		= $(QTDIR)/lib
-QMAKE_INCDIR_OPENGL	= /usr/X11R6/include
-QMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib64
+QMAKE_INCDIR_OPENGL	= /usr/X11/include
+QMAKE_LIBDIR_OPENGL	= /usr/X11/lib64
 
 QMAKE_LINK		= g++
 QMAKE_LINK_SHLIB	= g++
--- ./mkspecs/linux-g++/qmake.conf.vanilla	2005-10-08 09:37:47.000000000 +0200
+++ ./mkspecs/linux-g++/qmake.conf	2005-10-08 09:37:57.000000000 +0200
@@ -40,12 +40,12 @@
 
 QMAKE_INCDIR		=
 QMAKE_LIBDIR		=
-QMAKE_INCDIR_X11	= /usr/X11R6/include
-QMAKE_LIBDIR_X11	= /usr/X11R6/lib
+QMAKE_INCDIR_X11	= /usr/X11/include
+QMAKE_LIBDIR_X11	= /usr/X11/lib
 QMAKE_INCDIR_QT		= $(QTDIR)/include
 QMAKE_LIBDIR_QT		= $(QTDIR)/lib
-QMAKE_INCDIR_OPENGL	= /usr/X11R6/include
-QMAKE_LIBDIR_OPENGL	= /usr/X11R6/lib
+QMAKE_INCDIR_OPENGL	= /usr/X11/include
+QMAKE_LIBDIR_OPENGL	= /usr/X11/lib
 
 QMAKE_LINK		= g++
 QMAKE_LINK_SHLIB	= g++
